modules/openapi-generator/src/main/resources/go-server/service.mustache

```diff
@@ -23,7 +23,7 @@ func New{{classname}}Service() {{classname}}Servicer {
 {{#isDeprecated}}
 // Deprecated
 {{/isDeprecated}}
-func (s *{{classname}}Service) {{nickname}}(ctx context.Context{{#allParams}}, {{paramName}} {{dataType}}{{/allParams}}) (ImplResponse, error) {
+func (s *{{classname}}Service) {{nickname}}(ctx context.Context{{#allParams}}, {{paramName}} {{#isNullable}}*{{/isNullable}}{{dataType}}{{/allParams}}) (ImplResponse, error) {
```

Comment:
I don't `isNullable` is behaving as you expect it to here. The Pet Store Specs have many non-required params that would show this change but none of them are showing up in this diff.

Reply:
could you provide a specific example? are they marked as nullable?

Reply:
One specific example from `openapi-generator/samples/server/petstore/go-api-server/api/openapi.yaml` is the `booleanTest` query parameter in `deleteUser`.

```
- description: boolean query parameter
        explode: true
        in: query
        name: boolean_test
        required: false
        schema:
          type: boolean
        style: form
```

This should result in the the function signature for `deleteUser` having a `*bool` parameter for the function signature in `openapi-generator/samples/server/petstore/go-api-server/go/api_user_service.go`

There are numerous query parameters in that example file that have optional query parameters.

Reply:
object is not nullable, please see https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#schema-object